### ISSUE TITLE
Additional information for baseline algorithms

### ISSUE THREAD

**Issue by wsjeon** [rail-berkeley/softlearning#27]
Hi, 

Would you please share the information for the baselines, e.g., DDPG, TD3, PPO, with their hyper-parameter settings? (If you used open-source codes, would you please give me the link?)

While this repository only considers soft-learning algorithms, I think refactoring previous baseline algorithms in this framework might be quite useful. 

Thanks.

**Comment by azhou42** [comment]
For results in the main plots:
DDPG was ran using rllab's implementation https://github.com/rll/rllab/blob/master/rllab/algos/ddpg.py with similar hyperparameter settings as SAC when applicable. 

TD3 used the author's implementation https://github.com/sfujim/TD3 with the default hyperparameters.

PPO was ran using OpenAI baselines (before their refactor), while tuning the actor batch parameter for each env to make it work.

SQL was ran using basically the implementation in this repo (it was from a private repo that eventually got released as this) with the addition of the min Q trick that was also used in SAC. Like SAC, reward scale was tuned per env. 

I'll find and post the env specific parameters for PPO/SQL here in a bit.

**Comment by azhou42** [comment]
PPO batch sizes:
2048 actor batch size, 64 optimizer minibatch size by default
8192 actor batch for ant, 512 optimizer minibatch
16384 actor batch for humanoid-gym/rllab, 512 optimizer minibatch.

SQL common parameters:
    "kernel_n_particles": 16,
    "kernel_update_ratio": 0.5,
    "policy_lr": 0.001,
    "qf_lr": 0.001,
    "qf_target_n_particles": 16,
    "qf_target_update":  {
      "tau": 1.0,
      "update_interval": 1000
    }

SQL reward scales:
Ant: 300
Cheetah: 30
Hopper: 30
Humanoid-rllab: 100
Walker: 10

**Comment by hartikainen** [comment]
I'll close this for now. Feel free to reopen if there are any details missing.
